docs/41-managing-storage-via-R.md

```diff
@@ -0,0 +1,35 @@
+# Managing blob files in Azure Storage
+## Accessing your storage files through R
+Without installing Azure Storage Explorer or using the Azure Portal, users can access their resources through doAzureParallel wrapper functions around rAzureBatch's API calls.
+
+A container provides a grouping of a set of blobs. All blobs must be in a container. An account can contain an unlimited number of containers. A container can store an unlimited number of blobs. **Note** that the container name must be lowercase.
```

Comment:
container -> Storage container. Make sure to be consistent on this. It will get _really_ confusing later on once we introduce docker containers.

Reply:
Fixed